apps/omg_eth/test/support/deployer.ex

```diff
@@ -45,4 +45,14 @@ defmodule OMG.Eth.Deployer do
     Eth.deploy_contract(from, bytecode, [], [], opts)
     |> Eth.DevHelpers.deploy_sync!()
   end
+
+  def create_new(OMG.Eth.Eip712, path_project_root, from, opts) do
+    defaults = @tx_defaults |> Keyword.put(:gas, @gas_contract_token)
```

Comment:
use a dedicated gas attribute instead of `@gas_contract_token` (value can be the same ofc)